runtime/jcl/common/jclexception.c

```diff
@@ -372,3 +375,24 @@ getStackTrace(J9VMThread * vmThread, j9object_t * exceptionAddr, UDATA pruneCons
 
 	return result;
 }
+
+/**
+ * Check if the classloader name should be included in the stack trace.
+ *
+ * @param vm The Java VM.
+ * @param classLoader The classloader to check.
+ * @return true if the classloader name should be included, false otherwise.
+ */
+static BOOLEAN
+includeClassLoaderName(J9JavaVM *vm, J9ClassLoader *classLoader) {
+	/**
+	 * If the classloader is one of the built-in classloaders, don't include its name
+	 * in the stack trace.
+	 * To match the RI, make an exception if the classloader is the System/App classloader
+	 * and include its name.
+	 */
+	if ((NULL == classLoader) || (vm->extensionClassLoader == classLoader)) {
```

Comment:
What about systemClassLoader?

Reply:
What is the distinction between the System and App class loader? In the javadoc, it seems to regard the System and App class loader as the same thing.

Reply:
In the VM, systemClassLoader is the bootstrap loader.

Reply:
Ah, got it. Fixing.